theano/tensor/basic.py

```diff
@@ -6663,8 +6663,8 @@ def make_node(self, *shape):
         output.tag.nan_guard_mode_check = False
         return Apply(self, shape, [output])
 
-    def debug_perform(self, node, inputs, out_):
-        self.perform(node, inputs, out_)
+    def debug_perform(self, node, inputs, out_, params):
```

Comment:
There seems to be also a `debug_perform()` in a child class (`theano.gpuarray.basic_ops.GpuAllocEmpty#debug_perform`). I think it should be fixed too.